src/libexpr/eval.cc

```diff
@@ -465,6 +465,24 @@ EvalState::~EvalState()
 }
 
 
+void EvalState::requireExperimentalFeatureOnEvaluation(
+    const std::string & feature,
+    const std::optional<std::string> & fName,
```

Comment:
```suggestion
    std::string_view fName,
```

since I don't really see a situation where this could be unset.